sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryCompilationErrors.scala

```diff
@@ -203,13 +203,6 @@ private[sql] object QueryCompilationErrors extends QueryErrorsBase with Compilat
     invalidParameter("DOUBLE", "ewm", "alpha", invalidValue)
   }
 
-  def invalidStringParameter(
```

Comment:
Let's don't remove this, because it is supposed to use in the on-going PR: https://github.com/apache/spark/pull/46004